projects/valtimo/components/src/lib/components/carbon-list/carbon-list.component.html

```diff
@@ -100,3 +100,11 @@
     (click)="$event.stopPropagation(); data.callback(data.item)"
   ></i>
 </ng-template>
+
+<ng-template #defaultTemplate let-data="data">
```

Comment:
Are all viewtypeconverter supported? Why does this need to be coded in the template?